.travis.yml

```diff
@@ -23,48 +33,12 @@ matrix:
  include:
 # only one of these should have UPDATE_ERRATA set to "yes", otherwise we risk race conditions on pushing
    - env: LATEXMK="yes" UPDATE_ERRATA=""     UPDATE_NIGHTLIES=""     TARGETS=""
-     addons:
-       apt:
-         packages:
-           - latexmk
```

Comment:
Removing this section is wrong, unless you add a line to `install` to remove `latexmk` when `LATEXMK` is empty.  Otherwise none of the Travis jobs actually test that the makefile works fine without `latexmk`.

Reply:
I was beginning to wonder about that, thanks!